views/mixins/svg-elements.pug

```diff
@@ -1,13 +1,13 @@
 mixin icon(humanName, caption = humanName)
   svg.icon(width='28.5pt' height='28.5pt' viewbox='0 0 11 11' preserveaspectratio='xMidYMid')
     title= caption
-    - name = humanName.toLowerCase().replace(' ', '-')
+    - name = humanName.toLowerCase().replace(new RegExp(/\s/, 'g'), '-')
     - file = 'images/icons/' + name + '.svg#icon'
     clippath(id=`${name}ClipPath`)
       use(x='0' y='0' xlink:href=asset(file))
     use(x='0' y='0' xlink:href=asset(file) clip-path=`url(#${name}ClipPath)`)
 
 mixin sponsorLogo(humanName)
-  - name = humanName.toLowerCase().replace(' ', '-')
+  - name = humanName.toLowerCase().replace(new RegExp(/\s/, 'g'), '-').replace(new RegExp(/\./, 'g'), '')
```

Comment:
Are these intended as fail-safes?

Reply:
So up to now, only the first instance of a space would be replaced (which wasn't the intended behaviour). And I think replacing all whitespace with dashes makes sense